### DISCUSSION TITLE
PPOCRv4或v3如何使用openvino来做推理

### DISCUSSION THREAD

**Discussion by wxzwxz131** [PaddlePaddle/PaddleOCR#12184] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by wxzwxz131** [comment]
有没有大佬回答一下，因为现在v4版本都说自己是openvino推理，时间也是按这个来的

**Comment by WilliamQf-AI** [comment]
可以使用fastdeploy。如下设置使用openvino后端：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/40328063/b5da1a71-5734-4fc0-bd9d-5a254ef35a34)
。
具体实现可然后参考
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/40328063/4545c89f-d324-4411-bb28-902fdff278f9)
